catalyst_voices/apps/voices/lib/widgets/text_field/voices_password_text_field.dart

```diff
@@ -16,12 +16,15 @@ final class VoicesPasswordTextField extends StatelessWidget {
   /// Optional decoration. See [VoicesTextField] for more details.
   final VoicesTextFieldDecoration? decoration;
 
+  final ValueChanged<String>? onSubmitted;
```

Comment:
I would add it next to onChanged because the two are similar. Both the field and the constructor param.

Reply:
Got it, good point.